stdlib/http/src/main/java/org/ballerinalang/net/http/websocket/client/InitEndpoint.java

```diff
@@ -63,80 +61,35 @@ public static void initEndpoint(ObjectValue webSocketClient) {
         HttpWsConnectorFactory connectorFactory = HttpUtil.createHttpWsConnectionFactory();
         WebSocketClientConnectorConfig clientConnectorConfig = new WebSocketClientConnectorConfig(remoteUrl);
         String scheme = URI.create(remoteUrl).getScheme();
-        populateClientConnectorConfig(clientEndpointConfig, clientConnectorConfig, scheme);
+        WebSocketUtil.populateClientConnectorConfig(clientEndpointConfig, clientConnectorConfig, scheme);
         // Create the client connector.
         WebSocketClientConnector clientConnector = connectorFactory.createWsClientConnector(clientConnectorConfig);
-        WebSocketClientConnectorListener clientConnectorListener = new WebSocketClientConnectorListener();
-        // Add the client connector as a native data when the client is not a failover client
+        // Add the client connector as a native data
```

Comment:
```suggestion
        // Add the client connector as a native data
```
Some has the period and some don't.Please check all occurrences.